Nodejs/Product/Nodejs/Guids.cs

```diff
@@ -57,6 +58,7 @@ static class Guids
         public static readonly Guid NodejsDebugLanguage = new Guid(NodejsDebugLanguageString);
         public static readonly Guid NodejsNpmCmdSet = new Guid(NodejsNpmCmdSetString);
         public static readonly Guid TypeScriptDebugLanguage = new Guid(TypeScriptLanguageInfoString);
+        public static readonly Guid TypeScriptEditorFactory = new Guid(TypeScriptEditorFactoryString);

```

Comment:
I see TypeScriptEditorFactoryString and TypeScriptEditorFactory defined here, but what for? I don't see anywhere else in this change they are referenced.